kickstart.sh

```diff
@@ -304,8 +321,6 @@ git="$(which_cmd git)"
 NETDATA_SOURCE_DIR=
 if [ ! -z "${git}" -a -x "${git}" ]
 then
-    SOURCE_DST="/usr/src"
-
     [ ! -d "${SOURCE_DST}" ] && run ${sudo} mkdir -p "${SOURCE_DST}"
 
     if [ ! -d "${SOURCE_DST}/netdata.git" ]
```

Comment:
Why `if [ ! -d "${SOURCE_DST}/netdata.git" ]` but not just `netdata` is it really more viable? I think its pretty obvious source updates from git, but directory name with .git in name looks really alien.